src/view/ChewingEditor.cpp

```diff
@@ -128,6 +130,57 @@ void ChewingEditor::exportUserphrase(const QString& file)
     emit model_->exportUserphrase(exporter);
 }
 
+void ChewingEditor::showAbout()
+{
+/* Make about dialog shows like aboutQt() on Mac OS X.
+   So we have to prevent call twice on Mac OS X. */
+#ifdef Q_OS_MAC
+    static QPointer<QMessageBox> oldAboutBox;
+
+    if (oldAboutBox) {
+        oldAboutBox->show();
+        oldAboutBox->raise();
+        oldAboutBox->activeWindow();
+        return;
+    }
+#endif
+
+    QString caption = QString("<h3>%1</h3><p>%2</p>")
```

Comment:
I can't really say who's right here, but exposing control chars like `%1` for translators without programming exp. is not preferable, errors are expected.
But... this is a small project with only 2 lang. So there it is fine with current way...